src/Valitron/Validator.php

```diff
@@ -11,7 +11,7 @@
  * @author  Vance Lucas <vance@vancelucas.com>
  * @link    http://www.vancelucas.com/
  */
-class Validator
+class Validator implements ValidatorInterface
```

Comment:
Why add a whole new interface? Just put the const variables here. No need to load another file.